### ISSUE TITLE
Compile a list of tribes for user research (September 2020)

### ISSUE THREAD

**Issue by lfrohlich** [raft-tech/TANF-app#269]
**Description:**

We want to create a list of diverse tribes, who have had many different experiences and face many different challenges, to focus our research on.
The full data analysis will likely be delivered in a future sprint. 

**Criteria to fill (must fill 5 of these 9):**

- Different levels of accuracy within their data
- Different levels of communication with OFA
- Different levels of training
- Different levels of concern for data quality
- Not recently participated in a research study
- Different caseload sizes
- Different submission types (Sample vs Universal Data)
- Administrative Differences
- 477 Plan Accepted (Only Tribes)

**AC:**

- [x] Tribes are recommended to reach out to for Round 3.
- [x] STT attributes spreadsheet is updated with Tribe data

**Tasks:**

- [x]  Work with OFA and 18F to quantify any metrics related to the recruitment criteria to establish a baseline for narrowing down tribe selection.
- [x]  Pair findings from OFA interview notes, OFA interview synthesis, and 18F's "Understanding the Local Experience" to strengthen tribe selection
- [x]  Document recommended tribes in STT criteria document

**Supporting Documentation:**

User research findings:

- STT Spring 2020 ([Link](https://github.com/HHS/TANF-app/wiki/2020,-Spring:-Understanding-the-local-experience))
- OFA Interview Raw Notes ([Link](https://hhsgov.sharepoint.com/:x:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Bd21147f1-5fb3-472d-9acd-98fe2d7ffdb0%7D&action=view))
- OFA Interview Synthesis ([Link](https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B588709de-fe1f-49e5-8d52-cbc6e25fa9cd%7D&action=view))

**DoD:**

- [x]  Documentation: Draft list of tribes is shared with Lauren and Christine.
- [ ] @cmajel has reviewed and signed off
- [x] @lfrohlich has reviewed and signed off

**Deliverables:**

Recruitment Criteria and Down Selection Process Documentation: ([Link](https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B1a533a84-0000-423d-8274-c0767d9f471d%7D&action=view))
Updated STTs attributes spreadsheet with a list of Tribes: ([Link](https://hhsgov.sharepoint.com/:x:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Be83f84f1-71a0-459f-80a4-0d39bc250be9%7D&action=view))

**Comment by dk-ui** [comment]
@shubhi-raft @matthewjdoty Ready for Raft Review: A tribe has been selected based on our criteria. I also selected two states and three more tribes that also met the criteria as backups.

Recruitment Criteria and Down Selection Process Documentation: ([Link](https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B1a533a84-0000-423d-8274-c0767d9f471d%7D&action=view))
List of STTs: ([Link](https://hhsgov.sharepoint.com/:x:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Be83f84f1-71a0-459f-80a4-0d39bc250be9%7D&action=view))

**Comment by dk-ui** [comment]
This ticket was an add-on from #163. It included tribes to the existing list of recommended STTs. The only recruitment criteria that was added was 477 plan accepted and this is only applicable for tribes.

**Comment by cmajel** [comment]
@dk-ui love having all this detail in the sheet ❤️  - this will be a great resource for future research.

A few notes: 
* I count 13 participants in the `Final STTs to Interview for R3` tab. We had discussed limiting this round to 9 participants. Can you indicate which 9 we're reaching out to in the sheet?
* I see you listed `477 Plan Accepted (Only Tribes)` as a criteria to be met for recruitment of tribes. Can you say why that's a criteria for this round of research? 
* Thank you for moving this to the user research folder with restricted access 👍 . A note that this does include possible participant PII so we don't want to demo this to outside stakeholders.

**Comment by dk-ui** [comment]
@cmajel Thank you! I liked compiling all the information. If you see any columns that would be good to put in the spreadsheet for future criteria let me know.

Hopefully this helps:
- Good Point, I split them up and indicated what STTs are backups.
- We wanted 477 plan accepted as a criteria because we still don't understand the full implications of 477 on reporting requirements. In the "Criteria for the Next Round of Research" section of the [doc](https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B1a533a84-0000-423d-8274-c0767d9f471d%7D&action=view) it explains the rationale of including 477 in the recruitment criteria as "We want to understand how 477 plans impact the submission process" Let me know if you want to see that more specific or rephrased somehow.
- Yes, I have been meaning to separate the PII from the sheet. When it is separated, where should that sheet with the PII go so that it is secure?

**Comment by lfrohlich** [comment]
Thanks @dk-ui ! This is looking great. I updated the Tribal TANF Criteria on the Email to OFA Interviewees tab, although I realize now that may have just been a record of what you already asked OFA analysts. Specifically, I think it makes sense to have tools used: SFTP pilot/ currently using SFTP/ Sends Email for Data Submission. Some tribes were a part of the pilot but do not currently submit via SFTP, and I think those could be a good group to target. I'll note that Extranet is not the same as SFTP (pilot) so I'll make some updates there. 

I know we have discussed how we want to understand how 477 tribes have different processes, although I'm wondering now if that really should be this round. Let's discuss a bit more.
